File: helloworld_web_war/src/main/java/org/example/demo/controller/HomeController.java
package org.example.demo.controller;

import org.noear.solon.annotation.Controller;
import org.noear.solon.annotation.Mapping;
import org.noear.solon.core.handle.Context;

@Controller
public class HomeController {
    @Mapping("/")
    public void home(Context ctx) {
        //内部跳转到 /index.htm
        ctx.forward("/index.htm");
    }
}

File: helloworld_web_jar_kt/src/main/kotlin/com/example/demo/controller/HomeController.kt
package com.example.demo.controller

import org.noear.solon.annotation.Controller
import org.noear.solon.annotation.Mapping
import org.noear.solon.core.handle.Context


@Controller
class HomeController {
    @Mapping("/")
    fun home(ctx: Context) {
        //内部跳转到 /index.htm
        ctx.forward("/index.htm")
    }
}

